### PR TITLE
Signalfx exporter: add filtering option to copy_metrics translation rule

### PR DESCRIPTION
**Pull request by dmitryax**
**Description:**
Add a filtering option to "copy_metrics" translation rule. It makes it possible to copy datapoints with specific dimensions and skip others.

### CODE REVIEW

File: exporter/signalfxexporter/go.sum [hunk 1]
```diff
@@ -1367,6 +1367,7 @@ google.golang.org/protobuf v1.22.0/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2
 google.golang.org/protobuf v1.23.0 h1:4MY060fB1DLGMB/7MBTLnwQUY6+F09GEiz6SsrNqyzM=
 google.golang.org/protobuf v1.23.0/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2l/sGQquU=
 google.golang.org/protobuf v1.23.1-0.20200526195155-81db48ad09cc/go.mod h1:EGpADcykh3NcUnDUJcl1+ZksZNG86OlYog2l/sGQquU=
+google.golang.org/protobuf v1.24.0 h1:UhZDfRO8JRQru4/+LlLE0BRKGF8L+PICnvYZmx/fEGA=
```
Comment (bogdandrutu) [inline exporter/signalfxexporter/go.sum:1370]: Why this change?

Comment (dmitryax) [reply exporter/signalfxexporter/go.sum:1370]: got this line added after running `make test`

**Review by bogdandrutu — APPROVED** [review]
